datafusion-cli/src/main.rs

```diff
@@ -100,6 +102,21 @@ pub async fn main() {
     rl.save_history(".history").ok();
 }
 
+fn is_valid_data_dir(dir: String) -> std::result::Result<(), String> {
+    if Path::new(&dir).is_dir() {
+        Ok(())
+    } else {
+        Err(format!("Invalid data directory “{}”", dir))
```

Comment:
```suggestion
        Err(format!("Invalid data directory '{}'", dir))
```

The `'` quote is used in much of the rest of DataFusion so it might be useful to use that here, rather than smart quotes.